sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestData.scala

```diff
@@ -91,6 +91,12 @@ private[sql] trait SQLTestData { self =>
     df
   }
 
+  protected lazy val largeDecimals: DataFrame = {
```

Comment:
`SQLTestData` should only contain data that are widely used by many tests. Let's move the `largeDecimals` to its test case.